app/javascript/components/widget-controller.jsx

```diff
@@ -160,12 +163,16 @@ export class WidgetController extends React.Component {
 
     const fetchPolicy = currentWidgetId === 0 ? 'network-only' : 'cache-first';
 
+    const cityName = this.props.location.pathname.substring(1);
+
     return (
       <Query
         query={getWidgets}
-        variables={{ id: currentWidgetId }}
+        variables={{ id: currentWidgetId, loc: cityName }}
         fetchPolicy={fetchPolicy}
-        onCompleted={({ primaryLocation: { widgets } }) => {
+        onCompleted={response => {
+          const { location } = response;
```

Comment:
this is getting confusing because we have a `props.location` and now a `response.location` with two different shapes. Can we rename one of them?

Reply:
Will rename to props.cityName to match the gql query parameter name change to cityName. Seems clearer that way for sure.